bundles/org.platformio.eclipse.ide.installer/schema/org.platformio.eclipse.ide.installer.prerequisites.exsd

```diff
@@ -1,53 +1,64 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!-- Schema file written by PDE -->
-<!--
-	Copyright (c) 2021 PlatformIO and ArSysOp
-
-	Licensed under the Apache License, Version 2.0 (the "License");
-	you may not use this file except in compliance with the License.
-	You may obtain a copy of the License at
-
-		http://www.apache.org/licenses/LICENSE-2.0
-
-	Unless required by applicable law or agreed to in writing, software
-	distributed under the License is distributed on an "AS IS" BASIS,
-	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-	See the License for the specific language governing permissions and
-	limitations under the License.
-
-	SPDX-License-Identifier: Apache-2.0
-
-	Contributors:
-		ArSysOp - initial API and implementation
--->
 <schema targetNamespace="org.platformio.eclipse.ide.installer" xmlns="http://www.w3.org/2001/XMLSchema">
 <annotation>
       <appinfo>
          <meta.schema plugin="org.platformio.eclipse.ide.installer" id="org.platformio.eclipse.ide.installer.prerequisites" name="Installer Prerequisites"/>
       </appinfo>
+      <documentation>
+         
+      </documentation>
    </annotation>
 
    <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
       <complexType>
          <sequence>
-            <element ref="locator" minOccurs="1" maxOccurs="unbounded"/>
-            <element ref="architecture"/>
+            <element ref="architecture" minOccurs="0" maxOccurs="1"/>

```

Comment:
Do you use it? Does it really independent from "registry"?

Reply:
Yes, it is. We use this extension to define a way to fetch portable python distribution from PIO site, and there is no distribution provided for linux, so that's the case when we don't need this extension element implemented.

Reply:
Check org.platformio.eclipse.ide.installer.python.PythonDistribution for usage example.